newIDE/app/src/AssetStore/AssetsHome.js

```diff
@@ -219,33 +305,39 @@ export const AssetsHome = React.forwardRef<Props, AssetsHomeInterface>(
       },
     }));
 
-    const starterPacksTiles: Array<React.Node> = starterPacks.map(
-      (assetPack, index) => (
-        <PublicAssetPackTile
-          assetPack={assetPack}
-          onSelect={() => onPublicAssetPackSelection(assetPack)}
-          key={`${assetPack.tag}-${index}`}
-        />
+    const starterPacksTiles: Array<React.Node> = starterPacks
+      .map((assetPack, index) =>
+        !openedAssetCategory ||
```

Comment:
here you could probably do a `.filter()` with this logic first then map (to avoid a filter(Boolean)) (same below)

Reply:
Oh yeah, you're right